incubator/jasperserver/values.yaml

```diff
@@ -0,0 +1,70 @@
+## Bitnami jasperserver image version
+## ref: https://hub.docker.com/r/bitnami/jasperserver/tags/
+##
+image: bitnami/jasperserver
+
+## Specify a imagePullPolicy
+## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+##
+# imagePullPolicy:
+
+## User of the application
+## ref: https://github.com/bitnami/bitnami-docker-jasperserver#environment-variables
+##
+jasperserverUser: user
+
+## Application password
+## Defaults to a random 10-character alphanumeric string if not set
+## ref: https://github.com/bitnami/bitnami-docker-jasperserver#environment-variables
+jasperserverPassword: password
```

Comment:
According to the readme the password is a randomly generated string, but you have set it to `password` here. Please specify the param here as a comment, f.e. https://github.com/kubernetes/charts/blob/master/stable/testlink/values.yaml#L17-L21